src/JasperBus.Tests/Transports/LightningQueues/LQ_integration_specs.cs

```diff
@@ -23,6 +23,7 @@ public LQ_integration_specs()
 
             with(_ =>
             {
+                _.UseLightningQueuesTransport();
```

Comment:
I *think* I'd rather that we just have the LQ and InMemory transports both enabled out of the box, but let's talk over that one down the line